python/baseline/services.py

```diff
@@ -402,8 +331,11 @@ def predict(self, tokens, **kwargs):
         if not export_mapping:
             export_mapping = {'tokens': 'text'}
         label_field = kwargs.get('label', 'label')
-        tokens_seq, mxlen, mxwlen = self.batch_input(tokens)
-        self.set_vectorizer_lens(mxlen, mxwlen)
+        tokens_seq = self.batch_input(tokens)
+        for _, vectorizer in self.vectorizers.items():
```

Comment:
This should probably get moved into a method given that this loop is done in so many places.